### ISSUE TITLE
Effective way to scrape comments from Reddit redesign

### ISSUE THREAD

**Issue by stebbins** [stebbins/strigil#5]
Anyone have an idea?

The problem in particular is how Reddit's redesign handles timestamp data. Before the redesign, the raw timestamp was stored in the `data-timestamp` attribute on the high-level comment div. Now, it just stores the metadata for the timestamp i.e. "1 day ago"

**Comment by fig** [comment]
At the moment, it looks like the only other way is to follow the permalink where the timestamp is once again available in  the `data-timestamp` attribute, but that would be a lot of navigating for the web-driver.

Another option - if you think users might be wary of entering credentials, is to use the cookies. Instruct your users to log in to reddit and change the appropriate pref before firing up Strigil. The browser will remember the setting and serve legacy pages.  Skip execution of `Strigil::StrigilClient#login_and_make_legacy` if credentials are `empty?`

**Comment by stebbins** [comment]
I think that the credential aspect could be solved by your first idea i.e. not requiring them in the first place.

That's interesting, I didn't even think about following the permalink - personally, I'm fine with the increased webdriver navigation. Speed of extraction isn't necessarily my concern with this sort of thing, plus it could help solve https://github.com/stebbins/strigil/issues/2 if we can access permalinks for comments further back than 1k.

**Comment by stebbins** [comment]
Yeah, simply storing the permalinks is an excellent idea! We could then pass them off to a separate instance (or even Nokogiri) for extraction and storage. 

In terms of requests arousing suspicion... No idea :man_shrugging:
